docs/ProjectProperties.md

```diff
@@ -6,4 +6,4 @@ In addition to the standard set of MSBuild properties supported by Microsoft.NET
 Property name      | Meaning
 -------------------|--------------------------------------------------------------------------------------------
 IsShippingPackage  | When set to `true`, the package produced by from project is intended for use by customers. Defaults to  `false`, which means the package is intended for internal use only by Microsoft teams.
-IsAspNetCoreApp    | Set to `true` when the assembly is part of the [Microsoft.AspNetCore.App shared framework](./SharedFramework.md)
\ No newline at end of file
+IsAspNetCoreApp    | Set to `true` when the assembly is part of the [Microsoft.AspNetCore.App shared framework](./SharedFramework.md) and is not available as a NuGet package (unless IsShippingPackage is also set to `true`.)
```

Comment:
Property needs a new name if you really mean "and is not &hellip;". Isn't "not available as a NuGet package" true for all shared framework components these days?

IOW property name makes sense only so long it means the property ships in the shared framework (no caveats).

Reply:
> Isn't "not available as a NuGet package" true for all shared framework components these days?

Nope. Here are the exceptions: https://github.com/aspnet/AspNetCore/blob/f19e8da1daf2808f3bbb870141304b73e76a547a/eng/SharedFramework.Local.props#L10-L17